.github/workflows/gen_gbs_result_badge.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+##
+# SPDX-License-Identifier: LGPL-2.1-only
+# Copyright (C) 2024 Samsung Electronics
+#
+# @file gen_gbs_result_badge.py
+# @brief A tool for generating a github badge image in svg format
+# @author Gichan Jang <gichan2.jang@samsung.com>
+#
+import sys
+import os
+import json
+from pybadges import badge
+
+##
+# @brief Save the badge on the given path
+# @param[in] path A file path to save the svg file
+# @param[in] badge github badge svg file
+def save_badge (path, badge):
+    file = open(path, "w")
+    file.write(badge)
+    file.close()
+
+##
+# @brief Generate a github badge svg file representing daily build result
+# @param[in] html build result of arch.
+# @param[in] path A directory path to save the svg file
+def gen_build_result_badge(json_str, out_dir):
+    daily_result="success"
+    for arch in json_str:
+        result = json_str[arch]
+        daily_color = color= 'green'
+        if result == 'failure':
+          color = 'red'
+          daily_result = result
+          daily_color = 'red'
+
+        s = badge(left_text='test', right_text=result, right_color=color)
+        path = os.path.join(out_dir, arch + '_reuslt.svg')
+        save_badge (path, s)
+
+    s = badge(left_text='Daily build', right_text=result, right_color=color)
+    path = os.path.join(out_dir, 'daily_reuslt.svg')
+    save_badge (path, s)
+
+if __name__ == '__main__':
+    if len(sys.argv) < 3:
+        exit(1)
+
+    file = open(sys.argv[1], "r")
+
+    gen_build_result_badge (json.load(file), sys.argv[2])
```

Comment:
I think it would be good to modify this part as follows. 😄 
```
try:  
        with open(sys.argv[1], "r") as input_file:  
            data = json.load(input_file)  
            gen_build_result_badge(data, sys.argv[2])  
    except Exception as e:  
        print("An error occurred:", str(e))  
```

Reply:
Updated.